tests/unit/utils/parsers_test.py

```diff
@@ -804,6 +805,62 @@ def setUp(self):
 
 
 @skipIf(NO_MOCK, NO_MOCK_REASON)
+class DaemonMixInTestCase(LogSettingsParserTests):
```

Comment:
Does it really necessary to inherit from `LogSettingsParserTests`?
I see that you don't really need to do anything with logging... It would just execute all Runner tests once more.

Reply:
I was unsure about this. Then I'll just get rid of it. Thanks!